Consequences/ExternalMethods/NSI.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Runtime.InteropServices;
+using USACE.HEC.Consequences;
+using USACE.HEC.Geography;
+
+namespace USACE.HEC.ExternalMethods;
+
+public class NSI
+{
+  [UnmanagedCallersOnly(EntryPoint = "ReadNSI")]
+  public static int Read()
```

Comment:
I left this in as a test method and forgot to actually make it useful. For now I'll have a method that just reads from the NSI in case a user just has the Consequences project, and then in GeoConsequences I'll have a method that reads from the NSI and writes to shapefile